legend-saga/src/events.rs

```diff
@@ -184,6 +186,32 @@ impl PayloadEvent for LegendMissionsOngoingMissionEventPayload {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct CryptoRankingWinners {
+    pub user_id: String,
+    pub reward: f64,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct CompletedCryptoRanking {
+    pub wallet_address: String,
+    pub winners: Vec<CryptoRankingWinners>,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct LegendRankingsCryptoRankingFinishedPayload {
+    pub completed_crypto_rankings: Vec<CompletedCryptoRanking>,
```

Comment:
In the example is an object, not an array
```json
{
  "walletAddress": "0x0000000000000000000000000000000000001010",
  "winners": [
    {
      "userId": "507f191e810c19729de860ea",
      "reward": 0.0005
    },
    {
      "userId": "507f1f77bcf86cd799439011",
      "reward": 10.0
    }
  ]
}
```

Reply:
Yes, I meant that this would be the type of objects inside the array, corrected